src/renderer/modals/DisclaimerModal/index.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { PureComponent } from "react";
+import { DeviceModelId } from "@ledgerhq/devices";
+import { Trans, withTranslation, TFunction } from "react-i18next";
+import styled from "styled-components";
+import { FinalFirmware, OsuFirmware } from "@ledgerhq/live-common/lib/types/manager";
+import { urls } from "~/config/urls";
+import TrackPage from "~/renderer/analytics/TrackPage";
+import Track from "~/renderer/analytics/Track";
+import { openURL } from "~/renderer/linking";
+import Button from "~/renderer/components/Button";
+import Markdown, { Notes } from "~/renderer/components/Markdown";
+import Box from "~/renderer/components/Box";
+import { ModalStatus } from "~/renderer/screens/manager/FirmwareUpdate/types";
+import getCleanVersion from "~/renderer/screens/manager/FirmwareUpdate/getCleanVersion";
+import { Icons, Checkbox, Drawer, Alert, Flex, Link, Text } from "@ledgerhq/react-ui";
+
+type Props = {
+  t: TFunction;
+  status: ModalStatus;
+  firmware: {
+    osu: OsuFirmware;
+    final: FinalFirmware;
+  };
+  goToNextStep: () => void;
+  onClose: () => void;
+  modelId: DeviceModelId;
+};
+
+type State = any;
+
+class DisclaimerModal extends PureComponent<Props, State> {
```

Comment:
do we need to keep it as a class component? :thinking: 
maybe could be rewritten as a function one with memo

Reply:
Yep indeed, shouldn't we stick to "not touching the logic" unless necessary for rebranding work though ?

Reply:
In case some changes are made in the develop branches it will be easier to reconcile when we merge everything if the logic is modified as less as possible.